libp2p/src/main/kotlin/io/libp2p/pubsub/AbstractRouter.kt

```diff
@@ -23,7 +23,7 @@ const val DEFAULT_MAX_PUBSUB_MESSAGE_SIZE = 1 shl 20
 
 typealias PubsubMessageHandler = (PubsubMessage) -> CompletableFuture<ValidationResult>
 
-open class DefaultPubsubMessage(override val protobufMessage: Rpc.Message) : AbstractPubsubMessage() {
+class DefaultPubsubMessage(override val protobufMessage: Rpc.Message) : AbstractPubsubMessage() {
```

Comment:
Until this project has strict rules regarding members visibility I would suggest to leave them as is. I was using some of them for tweaking router behavior in side projects.